src/document.ts

```diff
@@ -21,7 +26,11 @@ export interface PrismicDocumentHeader<TypeEnum = string, LangEnum = string> {
 	lang: LangEnum;
 	alternate_languages: AlternateLanguage[];
 }
-
+/**
+ * Prismic Documents are instances of CustomTypes.
+ * 
+ * More details on Custom Types: {@link https://prismic.io/docs/core-concepts/custom-types}
+ */
```

Comment:
While this is true I think it's maybe more simple/less confusing to just say "A Prismic document served through REST API v2" with a link to https://prismic.io/docs/technologies/introduction-to-the-content-query-api

Reply:
^ 👍

(Side note: "CustomType" should be written as "Custom Type")